### ISSUE TITLE
Do we still need mpi in the setup.py?

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/spinalcordtoolbox#3458]
Currently, we have this:
~~~
        'mpi': [
            'mpich==3.2',
            'mpi4py==3.0.0',
~~~

still needed?

**Comment by joshuacwnewton** [comment]
• A file called `requirementsMPI.txt` was added in #1587 containing MPI dependencies.
• Those MPI dependencies were copied over to `setup.py` in #1996 (specifically: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/1996/commits/96bb59dc21b1d79bdc99db6cc741d0b55d30bfd0)
• MPI features were dropped in https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/e3804a891f8d39b6fd371268a09f0c110cd78478.
• As a follow-up, `requirementsMPI.txt` was removed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/c5bb97f78e9624b7409da3816af932e1054df1e2.
• Yet, the corresponding MPI dependencies in `setup.py` were not removed, even though they should have been.

tl;dr: Yes, we can get rid of them.
